src/virtual_dom/vtag.rs

```diff
@@ -546,7 +546,7 @@ impl<COMP: Component> PartialEq for VTag<COMP> {
             return false;
         }
 
-        if self.classes != other.classes {
+        if self.classes.iter().collect::<Vec<&String>>() != other.classes.iter().collect::<Vec<&String>>() {
```

Comment:
Looks like you can use this for inequality: https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.ne

Reply:
Ah, I tried a simple `!=` but that didn't work. Fixed!